### DISCUSSION TITLE
switching page and keeping last point of view

### DISCUSSION THREAD

**Discussion by WAAH1** [sumatrapdfreader/sumatrapdf#3525] [General]
hello everyone,

on sumatra pdf when i have 2 pages, and i zoom in on a specific area on page 1, then i go to page 2 and when i come back to page 1 I loose the zoom on the area i've focus 

is there any possibilities to keep the focus we have done on the page, then when we switch between page and we come back we are directly to the focus/zoom we have done

hope this is clear

thank you for your help !!

**Comment by GitHubRulesOK** [comment]
I don't think there is anything much like that possible in any lighter PDF viewer except Acrobat Reader DC, 
(Acrobat powered "Plug-in" in Edge seems NOT to have the same ability)
but I could easily be wrong (there are means to hyperlink to a scaled area but that's when editing a document).

In most/all lighter weight viewers when you use another page at same zoom you can "jump" back to previous page via ALT + Left Arrow or "back" icon/commands, but the zoom level will usually be as it is now current.

So its possible but AFAIK not a current SumatraPDF feature
